app/controllers/static_pages_controller.rb

```diff
@@ -4,4 +4,7 @@ def home
 
   def help
```

Comment:
nếu method body mà ko có nội dung gì, em viết lại như này cho ngắn gọn, tiết kiệm số dòng code nhé:
```suggestion
  def help; end
```
Em sửa tương tự cho 2 method còn lại ha.